src/css/custom.css

```diff
@@ -79,7 +79,7 @@ body {
 }
 
 .footer {
-  background-color: var(--ifm-color-primary);
+  background-color: var(--ifm-color-primary-darkest);
```

Comment:
made it darker for more contrast - wdyt?

Reply:
Nice - I dig it!